mvpa2/mappers/procrustean.py

```diff
@@ -208,3 +208,10 @@ def _train(self, source):
             d_r = np.linalg.norm(odatas[0] - res_r)/np.linalg.norm(odatas[0])
             debug('MAP_', "%s, residuals are forward: %g,"
                   " reverse: %g" % (repr(self), d_f, d_r))
+
+
+    def _compute_recon(self):
+        """For Procrustean mapper, inverse is transpose.
```

Comment:
Thanks for nudging me to write the test. I realized that I didn't handle scaling=True case.
Tests passed because they were testing correlation and/or zscoring data beforehand in case of real data.
Handled that and added a test now.